pkg/chunk/grpc/grpc_server_mock.go

```diff
@@ -0,0 +1,207 @@
+package grpc
+
+import (
+	"context"
+	"log"
+	"net"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
Same here

Reply:
In all the go files in the cortex code base, I see this grouping of imports I mean standards packages, other Github imports, and then the cortex project reference imports.